Sources/Testing/Support/Locked.swift

```diff
@@ -77,14 +76,10 @@ struct Locked<T>: @unchecked Sendable where T: Sendable {
     }
   }
 
-  var wrappedValue: T {
+  var rawValue: T {
```

Comment:
Should it now conform to `RawRepresentable`?

Reply:
Oops! I meant it to. Fixed.